modules/rational-ui.el

```diff
@@ -29,14 +28,18 @@ Use a plist with the same key names as accepted by `set-face-attribute'.")
 
 ;;;; Mode-Line
 
-;; Start up the modeline after initialization is finished
-(add-hook 'after-init-hook 'doom-modeline-init)
 
-;; Configure `doom-modeline'
-(customize-set-variable 'doom-modeline-height 15)
-(customize-set-variable 'doom-modeline-bar-width 6)
-(customize-set-variable 'doom-modeline-minor-modes t)
-(customize-set-variable 'doom-modeline-buffer-file-name-style 'truncate-except-project)
+(defcustom rational-ui-use-doom-modeline t
```

Comment:
I don't concur with setting a variable that controls an optional feature to `t` by default.

Reply:
> I don't concur with setting a variable that controls an optional feature to t by default.

thats to not break existing configs
i could change that i guess